src/Tribe/Aggregator/Cron.php

```diff
@@ -385,6 +386,75 @@ public function verify_child_record_creation() {
 		}
 	}
 
+	public function start_batch_pushing_records() {
+		if ( ! tribe( 'events-aggregator.main' )->is_service_active() ) {
+			return;
+		}
+
+		$records = Tribe__Events__Aggregator__Records::instance();
+
+		$query = $records->query( [
+			'post_status'    => Tribe__Events__Aggregator__Records::$status->pending,
+			'posts_per_page' => 15,
+			'order'          => 'ASC',
+			'meta_query'     => [
+				'origin-not-csv'               => [
+					'key'     => '_tribe_aggregator_origin',
+					'value'   => 'csv',
+					'compare' => '!=',
+				],
+				'batch-push-support-specified' => [
+					'key'     => '_tribe_aggregator_allow_batch_push',
+					'value'   => true,
+					'compare' => '=',
+				],
+				'batch-not-queued'             => [
+					'key'     => '_tribe_aggregator_batch_started',
+					'value'   => 'bug #23268',
+					'compare' => 'NOT EXISTS',
+				],
+			],
+			'after'          => '-4 hours',
+		] );
```

Comment:
:no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 22272ecd049fdc2f3a253f2220b56695</sub>